pkg/kubelet/dockertools/docker_manager.go

```diff
@@ -725,7 +725,7 @@ func (dm *DockerManager) runContainer(
 
 	userNsMode := ""
 	if opts.EnableHostUserNamespace {
-		userNsMode = "host"
+		userNsMode = namespaceModeHost
```

Comment:
SGTM. But these docker_manager codes will be removed, let @yujuhong decide how to address this PR:)